### ISSUE TITLE
Support WebP image format

### ISSUE THREAD

**Issue by raphaelschaad** [Flipboard/FLAnimatedImage#5]

**Comment by IanLuo** [comment]
is this feature included now?

**Comment by timonus** [comment]
https://github.com/facebook/FLAnimatedImage/commit/0cc5f18ad361dc8f590f63694b34db0e50368df2 adds support for this.

**Comment by lyonlai** [comment]
@timonus I'm having problems compiling the facebook's webp support. any clue? I'm getting undefined symbol for arm64

**Comment by timonus** [comment]
Yeah, I wasn't able to get it to work either. My guess would be to try older versions of the WebP framework.

**Comment by itaydressler** [comment]
Supporting webp would be awesome.

**Comment by atom2ueki** [comment]
try [YYWebImage](https://github.com/ibireme/YYWebImage) if you need webp support
